### ISSUE TITLE
[BUG] Some buttons/elements on Profile screen are not available to Screen reader users on Android. Use should be consistent across platforms

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5573]
**Describe the bug**
I was testing the Profile screen with TalkBack and Android and noticed on Android there were certain elements/buttons not available to the TalkBack user. When I tested the same buttons/elements on iOS, they had labels and were accessible by assistive technology. 

**To Reproduce**
See videos included in comments, one per platform. 

**Expected behavior**
All users should have access to the same features. 

**Screenshots**

![Image](https://github.com/user-attachments/assets/84ddeaea-2168-4aa5-9be6-6bbbd29fc354)

Should function the same across all platforms and devices.

**Additional context**

**Comment by acovher2** [comment]
Android Experience with TalkBack:

https://github.com/user-attachments/assets/34ea7317-2d1a-486b-93f6-830de69540ac

iOS with VoiceOver:

https://github.com/user-attachments/assets/d160e0a3-a609-4128-b8f7-7a44715e377a
